lib/auth0/api/v2/users.rb

```diff
@@ -43,13 +43,15 @@ def users(options = {})
         # The attribute connection is always mandatory but depending on the type of connection you are using there
         # could be others too. For instance, Auth0 DB Connections require email and password.
         # @see https://auth0.com/docs/api/v2#!/Users/post_users
-        # @param name [string] The user name.
-        # @param options [hash]
-        #   * :connection [string] The connection the user belongs to.
+        # @param connection [string]  The connection the user belongs to.
+        # @param options [hash] See https://auth0.com/docs/api/management/v2#!/Users/post_users for available options
         # @return [json] Returns the created user.
-        def create_user(name, options = {})
+        def create_user(connection, options = {})
+          if !connection.is_a?(String) || connection.is_empty?
+            raise Auth0::MissingParameter, 'Must supply a valid connection'
```

Comment:
Was it your￼ intention to change the exception type here? It's just in the previous commit it was `MissingConnection`

Reply:
yes, the missingconnection wasn't common enough to reuse, MissingParameter is sufficient